src/reducers/importText.ts

```diff
@@ -234,9 +253,9 @@ const importText = (state: State, { path, text, lastUpdated, preventSetCursor, r
     return reducerFlow([
       updateThoughts(imported),
       // set cusor to destination path's parent after collapse unless it's em or cusor set is prevented.
-      collapseContext({ deleteCursor: true, at: newDestinationPath }),
+      !noNeedOfCollapse ? collapseContext({ deleteCursor: true, at: newDestinationPath }) : null,
```

Comment:
Double negatives are harder to read, so it would be better to negate the value and call it something like `useDummy` or `importIntoDummy`.